.github/workflows/ci.yml

```diff
@@ -153,6 +153,15 @@ jobs:
           service_url: '${{ env.INTEGRATION_SERVICE_URL }}'
           requested_permissions: '{"scope":"integ","repositories":["github-token-minter"],"permissions":{"issues":"read"}}'
 
+      - id: 'mint-github-token-2'
```

Comment:
Does the existing action ned to be updated? I think the inputs are different right?

Reply:
Yes, this was just a test to make sure that it works conceptually. I'll need to land the `minty` action on main, then update the original action to look like `mint-token-2`.